### PR TITLE
chore: SAV-774 Add url tracking param info to readme

### PR DESCRIPTION
**Pull request by dstewart-trellis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ Implementing Savvy Widget is easy. Just copy-and-paste a few lines of code:
 </script>
 ```
 
+## Attribution & Tracking
+
+In order to get credit for your usage, you MUST set `data-url-tracking-params` to the value assigned by your partner success manager.
```
Comment (redice44) [inline README.md]: In this case it is `urlTrackingParams` in the `Savvy.configure`.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ Implementing Savvy Widget is easy. Just copy-and-paste a few lines of code:
...
+
+See https://docs.google.com/document/d/1GF1yxu8BMfpK30EJ4AJZ3lhVuX_6Ae0Cdi2LPXAiAqY for more information.
```
Comment (redice44) [inline README.md]: Let's make it a pretty link.

File: README.md (hunk 1 above)
Comment (dstewart-trellis) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (dstewart-trellis) [reply README.md]: Fixed.

**Review by redice44 — APPROVED** [review]

**Pull request merged**